Dockerfile

```diff
@@ -45,6 +55,9 @@ RUN wget https://github.com/eBay/NuRaft/archive/v${NURAFT_VERSION}.tar.gz && \
 # Set working directory
 WORKDIR /opt/tx-processor
 
+# Create Build Image
+FROM $BASE_IMAGE AS builder
```

Comment:
Do we need to use the external image since the base is defined above? Could this just be `FROM base as builder` or `FROM $IMAGE_VERSION as builder`?